test-requirements.txt

```diff
@@ -1,3 +1,4 @@
 coverage==4.5.4
 pytest-xdist
-pytest-ansible-units
+git+https://github.com/ansible-community/pytest-ansible-units.git ; python_version < '3.9'
```

Comment:
It looks like pytest-ansible just added support for 3.7. In this case, I think the best option is to align with what pytest-ansible supports, rather than using the unmaintained repo.